pkg/security/probe/discarder_monitor.go

```diff
@@ -28,7 +27,7 @@ type DiscarderStats struct {
 
 // DiscarderMonitor defines a discarder monitor
 type DiscarderMonitor struct {
-	client            *statsd.Client
+	probe             *Probe
```

Comment:
Do we really need the whole probe, why use `client statsd.ClientInterface` ?

Reply:
Will use the interface instead of probe